client/src/actions/embedding.js

```diff
@@ -31,8 +31,10 @@ export const layoutChoiceAction = (newLayoutChoice) => async (
   layout.
   */
   const { annoMatrix: prevAnnoMatrix } = getState();
+  const { obsCrossfilter: prevCrossfilter } = getState();
```

Comment:
nit: you could combine lines 33 & 34 into a single assignment, avoiding the extra call to getState().

Reply:
Good call! Done!